main.js

```diff
@@ -66,17 +87,55 @@ function setAPPListeners () {
  * Opens new window with index.html(Jitsi Meet is loaded in iframe there).
  */
 function createJitsiMeetWindow () {
-    jitsiMeetWindow = new BrowserWindow(jitsiMeetWindowOptions);
-    jitsiMeetWindow.loadURL(indexURL);
+  jitsiMeetWindow = new BrowserWindow(jitsiMeetWindowOptions);
+  jitsiMeetWindow.loadURL(indexURL);
+  microWindow = new BrowserWindow(microWindowOptions);
```

Comment:
yup, pretty sure that's what saul means...it would definitely be interesting to try that and see how it performs.  if it doesn't perform all that well, maybe we could try some other options like creating the peer connection but only encoding/sending the media when the main window loses focus (to avoid the encoding cost all the time)